custom/icds_reports/utils/aggregation.py

```diff
@@ -883,7 +885,7 @@ def aggregation_query(self):
           SELECT
             %(state_id)s AS state_id,
             %(month)s AS month,
-            case_load_ccs_record0 AS case_id,
+            DISTINCT case_load_ccs_record0 AS case_id,
```

Comment:
you've run this query on ICDS to confirm it works as expected for all states?

Reply:
not yet. how would you recommend testing that in a safe way

Reply:
This table isn't used for anything yet, so it can be run as needed from the command line. Generally I've done:

1. create the query/table
2. deploy the code
3. debug/iterate on the query for several states/months (there's a utility in the utils directory to recalculate this)
4. add to the aggregation script

If this needs to get deployed to be aggregated immediately for the p&l stuff for QA, then you could setup_release instead of deploying and debug there. (may need to figure out the migrations, but I think these are fine to run)

Reply:
I think fine to wait for now. I will test and get back to you